src/devices/Ft4222/README.md

```diff
@@ -11,7 +11,7 @@ You can find boards implementing this chip like on [bitWizard](http://bitwizard.
 ## Windows Requirements
 
 In order to have this FTDI board working and getting support for SPI, GPIO and I2C, you need to install in a path the ```LibFT4222.dll``` provided by FTDI Chip. You can find the latest version [here](https://www.ftdichip.com/Products/ICs/FT4222H.html).
-The version used to build this project is 1.4.2 and you can download it directly from FTDI [here](https://www.ftdichip.com/Support/SoftwareExamples/LibFT4222-v1.4.2.zip).
+The version used to build this project is 1.4.2 and you can download it directly from FTDI [here](https://www.ftdichip.com/Support/SoftwareExamples/LibFT4222-v1.4.3.zip).
```

Comment:
```suggestion
The version used to build this project is 1.4.3 and you can download it directly from FTDI [here](https://www.ftdichip.com/Support/SoftwareExamples/LibFT4222-v1.4.3.zip).
```

Reply:
might be worth to remove that number so we don't have to update it every time we update the link

Reply:
That's the reason I added the link on the latest update everywhere. I just took the occasion to update this working version to a version which is working even better :-)